assets/src/skate.d.ts

```diff
@@ -5,6 +5,7 @@ declare global {
     Appcues?: {
       identify: (userId: string) => void
       page: () => void
+      show: (id: string) => void
```

Comment:
I don't see this function called anywhere. What happens to the tests if it's not mocked?

Reply:
Yep, that is no longer needed.